chapter_05/p07_pairwise_swap.py

```diff
@@ -0,0 +1,18 @@
+def pairwise_swap(number):
+    mask_10 = 0xAAAAAAAA  # 32 bits
+    mask_01 = 0x55555555  # 32 bits
+    num_evn = number & mask_10
+    num_odd = number & mask_01
+    swp_num = (num_evn >> 1) | (num_odd << 1)
+    return swp_num
+
+
+def example():
```

Comment:
similarly, if you rename this function to start with `test_` and add assertions then it will be added to the test suite.